azure/resource_groups/app/template.json

```diff
@@ -32,9 +32,33 @@
       "type": "bool",
       "defaultValue": true
     },
+    "appServicePlanInstances": {
+      "type": "int",
+      "defaultValue": 2
+    },
+    "appServicePlanTier": {
+      "type": "string",
+      "defaultValue": "Standard"
+    },
+    "appServicePlanSize": {
+      "type": "string",
+      "defaultValue": "1"
+    },
+    "vspAppServicePlanInstances": {
+      "type": "int",
+      "defaultValue": 1
+    },
     "vspAppServiceDockerImage": {
       "type": "string"
     },
+    "vspAppServicePlanTier": {
+      "type": "string",
+      "defaultValue": "Standard"
+    },
+    "vspAppServicePlanSize": {
+      "type": "string",
+      "defaultValue": "2"
```

Comment:
Is there a reason for this to have a different default from the web app service plan?

Reply:
Nah, I guess I kinda saw the VSP as an ancilliary service from the main app, but there's a good chance it'll get just as much traffic as the main app, so happy to change it and review usage as we get real traffic